apigen.neon

```diff
@@ -1,6 +1,6 @@
 source: ./
 
-destination: docs.woothemes.com/images/sensei-apidocs
+destination: docs.woocommerce.com/images/sensei-apidocs
```

Comment:
That's the directory where the docs will be generated in. I manually copy it over to https://github.com/Automattic/docs.woothemes.com